### ISSUE TITLE
Refocus the "please add your..." CTA link when exiting profile edit mode

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#9774]
- open an empty field to edit it
- then cancel to exit edit mode

the focus should be on the "please add your..." CTA link but it is not. The functionality works for the Edit button, but not the "Add" button that is shown when the user does not have data on file.

![exit-add.gif](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/5d406002-66e5-4430-af0e-b9f20522f9f0)

**Comment by mshea0606** [comment]
@erikphansen is this still an issue?

**Comment by erikphansen** [comment]
@mshea0606 this can be validated
